### ISSUE TITLE
Viewable Person form needs to show contact info. etc.

### ISSUE THREAD

**Issue by VanAuken** [ReadyResponder/ReadyResponder#99]
**Person** form when accessed for viewing does not show contact information.  It only shows training information.  Please add contact information.

Currently in order to see a person's contact info, [phone, address, email, etc.], the person form has to be opened for editing.

**Comment by kgf** [comment]
This issue is because there were two methods of capturing contact info. One is deprecated, but the data has not been migrated to the newer system. We'll need to write a data migration as a first step to this, then the UI changes.

**Comment by houhoulis** [comment]
@kgf Could you explain what you mean by "data migration" ?  Also "we will need to write a data migration first" -- does that mean this issue is blocked?  Or can we add the user's info to the view form right now?

**Comment by houhoulis** [comment]
This issue is entirely separate from #60, but both are related to the Person view/edit form differences, and I'm hoping @pcai might look at one or both  😃

**Comment by pcai** [comment]
I looked at this briefly, but as of 55f56969175 I can't reproduce:

![capture](https://cloud.githubusercontent.com/assets/222655/17353081/65b482e0-590a-11e6-9b78-97ae76f737a7.PNG)

I suspect I have the modern version of this storage format that works correctly. If I can get some pointers on the old format I would be happy to take a crack at writing a migration. In the meanwhile I opened https://github.com/kgf/lims/issues/111 which prevents each update from duplicating `User#channels`.
